### PR TITLE
added index of agreement, updated documentation and header

### PR DESCRIPTION
**Pull request by sebhahn**
there should be a " " at both sides of the : in the numpy docstring for the parameter/return types.

### CODE REVIEW

File: pytesmo/metrics.py [hunk 1]
```diff
@@ -14,10 +14,10 @@
 #      names of its contributors may be used to endorse or promote products
 #      derived from this software without specific prior written permission.
 
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
-# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-# DISCLAIMED. IN NO EVENT SHALL VIENNA UNIVERSITY OF TECHNOLOGY,
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND # ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
```
Comment (cpaulik) [inline pytesmo/metrics.py]: One # too much

File: pytesmo/metrics.py [hunk 2]
```diff
@@ -26,71 +26,139 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-'''
```
Comment (cpaulik) [inline pytesmo/metrics.py]: Why did you remove that?

Comment (sebhahn) [inline pytesmo/metrics.py]: sorry, was a mistake..

File: pytesmo/metrics.py [hunk 3]
```diff
@@ -26,71 +26,139 @@
...
+    Normalized root-mean-square deviation (nRMSD).
 
+    Parameters
+    ----------
+    o : numpy.ndarray
+        Observations.
+    p : numpy.ndarray
+        Predictions.
+
+    Retruns
```
Comment (cpaulik) [inline pytesmo/metrics.py]: There is a typo in the ones you added

File: pytesmo/metrics.py (hunk 2 above)
Comment (sebhahn) [inline pytesmo/metrics.py]: wait you mean the authors, I think we should not have this as a docstring here

File: pytesmo/metrics.py [hunk 4]
```diff
@@ -26,71 +26,139 @@
...
+        Observations.
+    p : numpy.ndarray
+        Predictions.
 
+    Retruns
+    -------
+    mse : float
+        Mean square error (MSE).
+    mse_corr : float
+        Correlation componento of MSE.
```
Comment (cpaulik) [inline pytesmo/metrics.py]: Are you speaking Italian ;-)

File: pytesmo/metrics.py [hunk 5]
```diff
@@ -198,53 +270,84 @@ def pearson_conf(r, n, c=95):
...
         1D numpy array to calculate the metric
 
     Returns
     -------
-    rho : float
+    rho: float
         Spearman correlation coefficient
-    p-value : float
-        The two-sided p-value for a hypothesis test whose null hypothesis
+    p - value: float
```
Comment (cpaulik) [inline pytesmo/metrics.py]: Why the spaces between p-value and two-sided? They are both written without. e.g.: [p-value](http://en.wikipedia.org/wiki/P-value)

File: pytesmo/metrics.py [hunk 6]
```diff
@@ -198,53 +270,84 @@ def pearson_conf(r, n, c=95):
...
     """
-    Wrapper for scipy.stats.spearmanr
+    Wrapper for scipy.stats.spearmanr. Calculates a Spearman
+    rank-order correlation coefficient and the p-value to
+    test for non-correlation.
 
     Parameters
     ----------
-    x : numpy.array
+    x: numpy.array
```
Comment (cpaulik) [inline pytesmo/metrics.py]: These should be o and p then?

File: pytesmo/metrics.py [hunk 7]
```diff
@@ -99,89 +167,93 @@ def tcol_error(x, y, z):
...
     """
-    return sc_stats.pearsonr(x, y)
+    return sc_stats.pearsonr(o, p)
 
 
 def pearson_conf(r, n, c=95):
     """
     Calcalates the confidence interval of a given pearson
-    correlation coefficient using a fisher z-transform,
+    correlation coefficient using a fisher z - transform,
```
Comment (cpaulik) [inline pytesmo/metrics.py]: same here, why spaces?

File: pytesmo/metrics.py (hunk 5 above)
Comment (sebhahn) [inline pytesmo/metrics.py]: will be fixed with the next commit

File: pytesmo/metrics.py (hunk 7 above)
Comment (sebhahn) [inline pytesmo/metrics.py]: maybe an autopep issue?! fixed

**Comment by cpaulik** [comment]
Thanks for the documentation update. Regarding the " " around the ":". They are not necessary for sphinx to produce valid documentation so I really don't care a lot about that. But you are welcomo to fix them.

**Comment by sebhahn** [comment]
squash finished

**Comment by cpaulik** [comment]
I've cherry picked this into the develop branch. Thanks.
